WordPress/Classes/Utility/WPTableViewHandler.m

```diff
@@ -205,12 +205,12 @@ - (void)refreshTableViewPreservingOffset
 
 - (void)cacheRowHeight:(CGFloat)height forIndexPath:(NSIndexPath *)indexPath
 {
-    [self.cachedRowHeights setObject:@(height) forKey:[indexPath toString]];
+    [self.cachedRowHeights setObject:@(height) forKey:indexPath];
```

Comment:
`IndexPath` is both `Hashable` and `Equatable`. As long as you're not comparing them with `==`, it should be fine.